indica/operators/centrifugal_asymmetry.py

```diff
@@ -244,33 +262,38 @@ def __call__(  # type: ignore
             strictly_positive=True,
         )
 
+        input_check(
+            "mean_charge",
+            mean_charges,
+            DataArray,
+            ndim_to_check=3,
+            strictly_positive=True,
+        )
+
         toroidal_rotations = toroidal_rotations.sel(element=impurity)
 
         impurity_mass_int = ELEMENTS[impurity][1]
 
-        unified_atomic_mass_unit = 931.4941e6  # in eV/c^2
+        unified_atomic_mass_unit = spc.value(
+            "atomic mass unit-electron volt relationship"
+        )  # in eV/c^2
         impurity_mass = float(impurity_mass_int) * unified_atomic_mass_unit
 
-        mean_charge = ELEMENTS[impurity][0]
-
+        mean_charge = mean_charges.sel({"element": impurity}, drop=True)
         main_ion_mass_int = ELEMENTS[main_ion][1]
 
         main_ion_mass = float(main_ion_mass_int) * unified_atomic_mass_unit
 
         ion_temperature = ion_temperature.sel(element=impurity)
 
-        c = 3.0e8  # speed of light in m/s
-        toroidal_rotations /= c
-
         # mypy on the github CI suggests that * is in an Unsupported operand type
         # between float and DataArray, don't know how to fix yet so for now ignored
-        asymmetry_parameter = (
+        return (
```

Comment:
Same comment as earlier in the code, is there any reason to do the calculation in the `return` statement?

Reply:
Here the in-place modify method led to us mutating `toroidal_rotations` which is an input value and so might have side-effects.